integration-cli/daemon/daemon.go

```diff
@@ -718,7 +718,7 @@ func (d *Daemon) ReloadConfig() error {
 	errCh := make(chan error)
 	started := make(chan struct{})
 	go func() {
-		_, body, err := request.Get(d.Sock(), "/events")
+		_, body, err := request.DoOnHost(d.Sock(), "/events", request.Method(http.MethodGet))
```

Comment:
Guess this can be `request.SockRequest(` ? Then `DoOnHost` doesn't have to be exported